README.md

```diff
@@ -318,6 +318,54 @@ the `public` Postgres database schema. Please follow the steps below.
    database, and the data in them will remain intact, but the services will no
    longer reference those shards.
 
+## Merging Databases
+
+This section is for installations of On-Premise Depot that were done *after*
+the database shard migration listed above. If upgrade to a newer version of the
+On-Premise Depot, you will be required to also merge databases into
+the `builder` Postgres database. Please follow the steps below.
+
+### Pre-requisites
+1. The password to your Postgres database. By default, this is located at
+   `/hab/svc/builder-datastore/config/pwfile`
+1. A fresh backup of the two databases present in the On-Premise Depot,
+   `builder_sessionsrv` and `builder_originsrv`. You can create such a backup
+   with `pg_dump`:
+
+   ```shell
+   PGPASSWORD=$(sudo cat /hab/svc/builder-datastore/config/pwfile) hab pkg exec core/postgresql pg_dump -h 127.0.0.1 -p 5432 -U hab builder_originsrv > builder-originsrv.sql
+   ```
+
+### Migration
+1. Uninstall existing services by running `sudo -E ./uninstall.sh`
+1. Install new services by running `./install.sh`
+1. Optionally, if you want to be extra sure that you're in a good spot to perform the
```

Comment:
Add a step to immediately stop the API service so that there is no incoming traffic adding data to the fresh builder db